sponsorship_compassion/models/contracts.py

```diff
@@ -537,8 +537,9 @@ def contract_active(self):
 
         # Update sponsorships on partner
         partners = self.mapped('partner_id') | self.mapped('correspondant_id')
-        partners._compute_has_sponsorships()
-
+        partners.number_sponsorships += 1

```

Comment:
There could be more than 1 partner, so you must add a loop `for partner in partners:`